staging/src/k8s.io/apiserver/pkg/server/genericapiserver_graceful_termination_test.go

```diff
@@ -727,14 +741,24 @@ func (a *fakeAudit) Run(stopCh <-chan struct{}) error {
 }
 
 func (a *fakeAudit) Shutdown() {
-	// TODO: uncomment it in https://github.com/kubernetes/kubernetes/pull/110026
-	// <-a.shutdownCh
+	<-a.shutdownCh
```

Comment:
mostly curiosity, I expected shutdown to actually close the channel , not blocking waiting for it to be closed